schema.graphql

```diff
@@ -297,18 +413,36 @@ type TicketNodeEdge {
 
 scalar UUID
 
+type UpdateAccount {
+  success: Boolean
+  errors: [GQLErrorUnion]
+}
+
 type UserNode implements Node {
   lastLogin: DateTime
   isSuperuser: Boolean!
-  username: String!
   firstName: String!
   lastName: String!
-  email: String!
   isStaff: Boolean!
   isActive: Boolean!
   dateJoined: DateTime!
   id: ID!
+  email: String!
   bookings(offset: Int, before: String, after: String, first: Int, last: Int, bookingReference: UUID, user: ID, performance: ID, status: String, id: ID): BookingNodeConnection!
```

Comment:
status here should be of the BookingStatus enum type

Reply:
For all other filtering I have been doing, it is using the enum type, not a string. I assumed this would be how it works because one a booking node, the status is of the enum type, and not a string.. bit of a double standard?

i.e. 
```
bookings(performance: $performanceID, status: INPROGRESS) {
      edges {
        node {
          ...DetailedBookingDetails
        }
      }
    }
```

If this isn't possible and we have to use strings instead I guess its fine, just a bit whack switching between the gql enum types and strings

Re your point number 2, I think this is probably expected because this is not a validation error (i.e. its more a syntax error with the query we submitted to graphql) so I think thats fine.

Reply:
Ah I see what you are saying, but when you actually get the response of the enum it should be like this: 
```
{
  "node": {
    "status": "INPROGRESS"
  }
}
```
i.e. it will also be a string